frontend_lib/test/component/IconButton.spec.js

```diff
@@ -17,37 +17,36 @@ describe('<IconButton />', () => {
 
   const wrapper = shallow(
     <IconButton
-      icon={props.icon}
-      className={props.className}
-      onClick={props.onClick}
-      disabled={props.disabled}
-      text={props.text}
-      style={props.style}
+      {...props}
     />
   )
 
-  it(`should display " ${props.text}"`, () =>
-    // Is it normal to have an space before the text ?
-    expect(wrapper.find(`.${props.className}`)).to.have.text().equal(' ' + props.text)
-  )
-
-  it(`should have the class "${props.className}"`, () =>
-    expect(wrapper.find(`.${props.className}`)).to.have.lengthOf(1)
-  )
-
-  it(`should display its text in color ${props.style.color}`, () =>
-    expect(wrapper.find(`.${props.className}`).prop('style')).to.deep.equal(props.style)
-  )
-
-  it(`should be set disabled to : ${props.disabled}`, () =>
-    expect(wrapper.find(`.${props.className}`).prop('disabled')).to.equal(props.disabled)
-  )
-
-  it(`should have the icon : "${props.icon}"`, () =>
-    expect(wrapper.find(`.fa-${props.icon}`)).to.have.lengthOf(1)
-  )
-
-  it(`should have the good onClick function"`, () =>
-    expect(wrapper.find(`.${props.className}`).prop('onClick')).to.equal(props.onClick)
-  )
+  describe('Static design test', () => {
+    it(`should display " ${props.text}"`, () =>
+      // Is it normal to have a space before the text ?
```

Comment:
use the proper comment rules:
`// INFO - GM - 2019-07-31 - Is it normal to have a space before the text ?`
Unit tests are meant to check that the component works as it's supposed to. If your test show something unexpected while the test is checking exactly what the component is supposed to do, then the issue probably comes from the component.
You are allowed to debug it in that case.